Taarafo.Core/Services/Foundations/PostImpressions/PostImpressionService.Validations.cs

```diff
@@ -33,6 +35,45 @@ private void ValidatePostImpressionOnAdd(PostImpression postImpression)
                 (Rule: IsNotRecent(postImpression.CreatedDate), Parameter: nameof(PostImpression.CreatedDate)));
         }
 
+        private void ValidatePostImpressionOnModify(PostImpression postImpression)
+        {
+            ValidatePostImpressionIsNotNull(postImpression);
+
+            Validate(
+            (Rule: IsInvalid(postImpression.PostId), Parameter: nameof(PostImpression.PostId)),
```

Comment:
Should TAB in one more for all the rules.

Reply:
Done